framework/py/flwr/serverapp/strategy.py

```diff
@@ -162,88 +162,121 @@ def start(
         evaluate_config : ConfigRecord, optional
             Configuration to be sent to nodes during evaluation rounds.
             If unset, an empty ConfigRecord will be used.
-        central_eval_fn : Callable[[int, ArrayRecord], MetricRecord], optional
-            Optional function for centralized evaluation. Takes server round number
-            and array record, returns a MetricRecord. If provided, will be called
-            before the first round and after each round. Defaults to None.
+        global_evaluate_fn : Callable[[int, ArrayRecord], MetricRecord], optional
+            Optional function for centralized evaluation of the global model. Takes
+            server round number and array record, returns a MetricRecord. If provided,
+            will be called before the first round and after each round. Defaults to
+            None.
 
         Returns
         -------
-        StrategyResults
-            Results containing training metrics, evaluation metrics, centralized
-            evaluation metrics (if provided), and final model arrays from all rounds.
+        Results
+            Results containing final model arrays and also training metrics, evaluation
+            metrics and global evaluation metrics (if provided) from all rounds.
         """
         log(INFO, "Starting %s strategy:", self.__class__.__name__)
         log_strategy_start_info(
             num_rounds, initial_arrays, train_config, evaluate_config
         )
-        self.summary()
-        log(INFO, "")
+        summary = self.summary()
+        log(INFO, "%s\n", summary)
```

Comment:
the `summary` method logs and returns none. I think it's better to keep it as it was